tests/qxmppiceconnection/tst_qxmppiceconnection.cpp

```diff
@@ -19,6 +20,8 @@ private slots:
 
 void tst_QXmppIceConnection::testBind()
 {
+    SKIP_IF_INTEGRATION_TESTS_DISABLED();
```

Comment:
just a note: `SKIP_IF_INTEGRATION_TESTS_DISABLED()` doesn't need an extra `;` at the end